pkg/connector/client.go

```diff
@@ -2,27 +2,46 @@ package connector
 
 import (
 	"context"
+	"errors"
 	"fmt"
-	"reflect"
+	"strconv"
+	"time"
 
 	"github.com/rs/zerolog"
 
+	"go.mau.fi/mautrix-meta/config"
 	"go.mau.fi/mautrix-meta/messagix"
 	"go.mau.fi/mautrix-meta/messagix/cookies"
+
 	"go.mau.fi/mautrix-meta/messagix/table"
 	"go.mau.fi/mautrix-meta/messagix/types"
 
+	"go.mau.fi/mautrix-meta/pkg/connector/msgconv"
+
 	"maunium.net/go/mautrix/bridge/status"
 	"maunium.net/go/mautrix/bridgev2"
+	"maunium.net/go/mautrix/bridgev2/database"
 	"maunium.net/go/mautrix/bridgev2/networkid"
+	"maunium.net/go/mautrix/event"
+
+	metaTypes "go.mau.fi/mautrix-meta/messagix/types"
 )
 
+type metaEvent struct {
+	context context.Context
```

Comment:
Oh, another thing I wanted to ask about: I store a Context in this struct so that it can be passed into the `incomingEvents` channel, which is picked up by `go handleMetaEventLoop()` -- 

I thought a Context would be desirable for cancellation etc. in the future, especially since it is a goroutine, but I realize that you're not supposed to put Contexts in a struct... is there a better solution in this case?